src/Translator.php

```diff
@@ -105,7 +105,8 @@ protected function execute($uri, array $parameters)
         curl_setopt($this->handler, CURLOPT_URL, static::BASE_URL . $uri);
         curl_setopt($this->handler, CURLOPT_POST, true);
         curl_setopt($this->handler, CURLOPT_POSTFIELDS, http_build_query($parameters));
-        
+		curl_setopt($this->handler, CURLOPT_SSL_VERIFYPEER, false);
```

Comment:
Fine by me.
But this is just translations, IMO.
I will Patch this later as by your feedback.